Gopkg.toml

```diff
@@ -0,0 +1,82 @@
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  name = "github.com/DATA-DOG/go-sqlmock"
+  version = "1.3.0"
+
+[[constraint]]
+  name = "github.com/blang/semver"
+  version = "3.5.1"
+
+[[constraint]]
+  name = "github.com/coocood/freecache"
+  version = "1.0.0"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/erikstmartin/go-testdb"
+
+[[constraint]]
+  branch = "master"
```

Comment:
should we consider locking down versions of the some of the dependencies that we are using from master?

Reply:
Yes, even if just freezing them to the commit hash for whatever master currently is. We'll also need to set up Jenkins autodeploy to use dep so it doesn't break with removing glide. There's no urgency on all of this, I more just wanted to see how much work it would be to switch over and it was super simple.